oci/private/pull.bzl

```diff
@@ -236,7 +236,10 @@ def _download_manifest(rctx, state, identifier, output):
     bytes = None
     manifest = None
 
-    result = _download(rctx, state, identifier, output, "manifests", allow_fail = True)
```

Comment:
we already have a fallback that sets this header. Did the fallback strategy didn't work for you?

https://github.com/bazel-contrib/rules_oci/pull/322/files#diff-4649111173e0628fda68b8b26f4d67e2ea6ae47454f39c02e3393f41170f2709R268

Reply:
@thesayyn You are right, what problems I encountered seems actually a local cache issue.